registry.json

```
@@ -769,6 +769,22 @@
             ]
         },
         {
+            "name": "detailed-usage-table",
```

Biome (2.1.2)

[error] The key name was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)